src/styles/watch/stopwatch.scss

```diff
@@ -0,0 +1,43 @@
+.stopwatch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
This is relevant to positions too, don’t use it on block itself
<img width="801" alt="image" src="https://user-images.githubusercontent.com/118751034/213766711-9194a872-95f4-412f-962f-e5ad5774f57a.png">